ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/search_tree/inputs/SearchTreeInput.java

```diff
@@ -124,14 +133,20 @@ public SearchTreeInputBuilder withToolProvider(ToolProvider toolProvider) {
             return this;
         }
 
+        public SearchTreeInputBuilder withCrac(Crac crac) {
+            this.crac = crac;
+            return this;
+        }
```

Comment:
you only need the outage instant, so replace this with an "withOutageInstant" method